tasks/test.js

```diff
@@ -75,45 +75,19 @@ module.exports = function (grunt) {
 
   grunt.registerTask('test:ui', [
     'checkPlugins',
-    'run:testEsServer',
-    'run:funcTestServer',
-    'functional_test_runner:functional',
-    'stop:testEsServer',
-    'stop:funcTestServer'
+    'run:functionalTests',
   ]);
 
   grunt.registerTask('test:uiRelease', [
     'checkPlugins',
-    'run:testEsServer',
-    'run:ossDistFuncTestServer',
-    'functional_test_runner:functional',
-    'stop:testEsServer',
-    'stop:ossDistFuncTestServer'
+    'run:functionalTestsRelease',
   ]);
 
   grunt.registerTask('test:ui:server', [
     'checkPlugins',
-    'run:testEsServer',
-    'run:devFuncTestServer:keepalive'
+    'run:functionalTestsDevServer',
   ]);
 
-  grunt.registerTask('test:api', [
-    'run:testEsServer',
-    'run:apiTestServer',
-    'functional_test_runner:apiIntegration',
```

Comment:
How do I make sure that it's no longer being used? It's kind of referenced strangely. Isn't it used in tasks/jenkins.js line 41? But I don't think that line is used anywhere, so it can be deleted.

Reply:
So, grunt works via "named tasks" (that the config folder and [load-grunt-config](https://github.com/firstandthird/load-grunt-config) kind of obfuscates) so if we were still relying on the `functional_test_runner` grunt tasks, we'd see a reference to `functiona_test_runner:` somewhere in these grunt tasks, which we no longer do.